models/snapchat__utm_report.sql

```diff
@@ -1,4 +1,4 @@
-with report as (
+with ad_hourly as (
```

Comment:
I see in this model we are attributing campaigns and ad_squads to the utm ads. I thought this could cause duplicate results? Should we be removing those fields from this model as well as the ad report?